application/modules/home/views/side_ci.php

```diff
@@ -0,0 +1 @@
+<h3 class="box-title"><i class="fa fa-building"></i><?php echo $exhibition['name']; ?> に加盟している会社情報編集</h3>
```

Comment:
__[code_sniffer]__
* File "side_ci.php" should be saved with Unicode (UTF-8) encoding, but it did not successfully pass the W3C test.
* Any char before the opening PHP tag is prohibited. Please remove newline or indentation before the opening PHP tag.
* Missing file doc comment
* No comment block marks the end of file instead of the closing PHP tag. Please add a comment block containing only "End of file side_ci.php".
* No comment block marks the end of file instead of the closing PHP tag. Please add a comment block containing only "Location: ./modules/home/views/side_ci.php".
* Any char after the closing PHP tag is prohibited. Please removes newline or spaces after the closing PHP tag.

<!-- Comment by [SideCI](https://sideci.com) -->